.prow.yaml

```diff
@@ -376,7 +376,7 @@ presubmits:
           limits:
             memory: 4Gi
 
-  - name: pre-kubermatic-e2e-aws-coreos-1.18
+  - name: pre-kubermatic-e2e-aws-flatcar-1.18
```

Comment:
"error":"error validating .prow.yaml: failed to validate .prow.yaml: duplicated presubmit job: pre-kubermatic-e2e-aws-flatcar-1.18"